packages/styles/scss/components/breadcrumb/_breadcrumb.scss

```diff
@@ -0,0 +1,150 @@
+//
+// Copyright IBM Corp. 2018, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@use '../../breakpoint' as *;
+@use '../../config' as *;
+@use '../../motion' as *;
+@use '../../spacing' as *;
+@use '../../theme' as *;
+@use '../../type' as *;
+@use '../../utilities/component-reset';
+@use '../../utilities/convert';
```

Comment:
Did we decide to `as *` this as well? Would impact the usage below to no longer have the namespace
```diff
-@use '../../utilities/convert';
+@use '../../utilities/convert' as *;

// usage
-convert.rem(1px)
+rem(1px)
```